src/main/java/core/basesyntax/service/CustomException.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+public class CustomException extends RuntimeException {
```

Comment:
CustomException is not a very informative name. I suggest RegistrationException or something similar.